engine/src/main/java/org/terasology/engine/physics/Physics.java

```diff
@@ -69,4 +72,41 @@
      *         given collision groups.
      */
     List<EntityRef> scanArea(AABBf area, Iterable<CollisionGroup> collisionFilter);
+
+    /**
+     * The epsilon value is the value that is considered to be so small that it
+     * could just as well be zero. Objects that are closer together than this
+     * value are assumes to be colliding.
+     *
+     * @return The simulation epsilon.
+     */
+    float getEpsilon();
+
+    /**
+     * Get the character collider for the given entity. Will create a new
+     * CharacterCollider if non exists and return that one.
+     *
+     * @param entity
+     * @return true if the given entity has a CharacterCollider associated to it.
+     */
+    CharacterCollider getCharacterCollider(EntityRef entity);
+
+    /**
+     * Removes the CharacterCollider associated with the given entity from the
+     * physics engine. The collider object of this entity will no longer be
+     * valid.
```

Comment:
Does it also remove the CharacterCollider component from the entity, or only invalidate it in the physics engine?

Reply:
probably can do this with another entity attached to the player then having this special edge case introduced?